packages/upset/src/components/ElementView/generatePlotSpec.ts

```diff
@@ -38,13 +72,39 @@ export function createScatterplotSpec(
   };
 }
 
-export function createScatterplotRow(specs: Scatterplot[]) {
+/**
+ * Creates the spec for multiple scatterplots containing data from multiple subsets.
+ * @param specs       Scatterplot objects with x and y attributes.
+ * @param selection   The current brush selection, if extant
+ * @param selectColor The color to use for brushed points
+ * @returns The Vega-Lite spec for the scatterplots.
+ */
+export function createScatterplotRow(
+  specs: Scatterplot[],
+  selection: ElementSelection | undefined,
+  selectColor: string,
+): VisualizationSpec[] {
   return specs.map((s) => ({
     width: 200,
     height: 200,
     mark: {
       type: 'point',
     },
+    // We only add the 'params' field if this object has a selection OR if there is no selection
+    // This works around a Vega bug where providing the value field to a param doesn't always work in concatenated plots
+    ...((!selection || (selection && convertSelection(s, selection))) && {
```

Comment:
"If the object has a selection, or if there is none" does this mean that you can't select on this scatterplot if there is another selection on another plot? I don't remember seeing this behavior in the UI testing

Reply:
Discussed in meeting